### ISSUE TITLE
4.6.0 breaks MySQL link

### ISSUE THREAD

**Issue by Berserkir-Wolf** [Ombi-app/Ombi#4393]
**Describe the bug**
Users with 4.6.0 and MySQL are getting a TypeLoadException with the MySQL interface.

**To Reproduce**
1. Update to 4.6.0
2. Run Ombi.
3. See error

**Expected behavior**
Ombi runs normally.

**Screenshots**

**Logs (Logs directory where Ombi is located)**
[Error in pastebin.](https://pastebin.com/jsK0xzGu)

**Desktop (please complete the following information):**
 - OS: Multiple, tested with ubuntu, docker (linuxserver) and windows.

**Ombi Version (please complete the following information):**
 - Version 4.6.0
- Media Server (not applicable, occurs in blank copy also)
- Database Type: MySQL

**Comment by Berserkir-Wolf** [comment]
Ties directly to .net 6 migration in #4390

**Comment by Berserkir-Wolf** [comment]
Seems to be related to charset ptions, apparently the way we were doing it is no longer supported, as per [this comment](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/issues/1435#issuecomment-846309406)

**Comment by Berserkir-Wolf** [comment]
Examples for migrations:
```c#
migrationBuilder.AlterDatabase()
    .Annotation("MySql:CharSetDelegation", DelegationModes.ApplyToColumns)
    .OldAnnotation("MySql:CharSet", "utf8mb4");
```

Potentially have to adjust everywhere a charset is mentioned, option now deprecated.

**Comment by tidusjar** [comment]
Should be fixed in v4.6.1
